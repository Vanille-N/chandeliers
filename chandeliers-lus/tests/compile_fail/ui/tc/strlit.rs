//! Bad literal.
chandeliers_lus::decl! {
    const S: int = "foobar";
}

fn main() {}
