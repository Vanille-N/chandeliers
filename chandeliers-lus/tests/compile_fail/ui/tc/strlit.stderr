error: Lustre only accepts literals of type int, float, or bool
 --> tests/fail/ui/tc/strlit.rs:2:20
  |
2 |     const S: int = "foobar";
  |                    ^^^^^^^^
